### ISSUE TITLE
PiT 24.4: form description is filled incorrectly, making test fail

### ISSUE THREAD

**Issue by manolo** [vaadin/form-filler-demo#17]
When selecting template 003, the form description is filled with the text 
`This is an invoice of an order for the project 'Vaadin AI Form Filler' providing some hardware`
instead of
`Vaadin AI Form Filler` 
see this [video](https://recordit.co/p86zhldxH8)

Test fail with the error 

```
org.opentest4j.AssertionFailedError: expected: <Vaadin AI Form Filler> but was: <This is an invoice of an order for the project 'Vaadin AI Form Filler' providing some hardware>
...
	at com.example.application.FormFillerTextIT.verifyFormValues(FormFillerTextIT.java:134)
	at com.example.application.FormFillerTextIT.formFill_chooseText1_fillsForm(FormFillerTextIT.java:76)

```

To reproduce checkout the project and set `form-filler-addon` version to `1.0-SNAPSHOT` and run the app

**Comment by czp13** [comment]
The older versions would not work because the old model was deprecated, so the newest form-filler-addon version is needed. (but the PIT tests you did were using the new model). Otherwise, with the old version, we would have this exception (and not assert exceptions)
```
com.theokanning.openai.OpenAiHttpException: The model `text-davinci-003` has been deprecated, learn more here: https://platform.openai.com/docs/deprecations
	at com.theokanning.openai.service.OpenAiService.execute(OpenAiService.java:330) ~[service-0.15.0.jar:na]
```

As we discussed on Slack, this is the problem: we removed the deprecated model (text-davinci-003) and added a new model, the AI model (gpt-3.5-turbo-instruct), causing the responses to be slightly different, which makes the IT test assertions fail.

So, we need to have a PR that fixes the assertion texts to the new ones (hopefully just a handful of differences and other things should work, as when I created that PR the form was filled nicely based on my tests).
